cli/tsc/99_main_compiler.js

```diff
@@ -1358,7 +1358,6 @@ delete Object.prototype.__proto__;
     "BufferEncoding",
     "BufferConstructor",
     "WithImplicitCoercion",
-    "Buffer",
```

Comment:
This should not be removed, it's the Node global

Reply:
Whoops. Reverted.